frontend/ui/widget/toggleswitch.lua

```diff
@@ -73,7 +73,15 @@ function ToggleSwitch:init()
         w = item_width,
         h = item_height,
     }
+    local diff_one_step = item_width - (frame_inner_width / self.n_pos - 2*item_border_size)
```

Comment:
Took me a bit of time to understand were that was coming from - I just had to look above :)

So, may be, put that nearer to its counterpart:
```diff
-local item_width = math.ceil(frame_inner_width / self.n_pos - 2*item_border_size)
+-- We'll need to adjust items width and distribute the accumulated fractional part to some
+-- of them for proper visual alignment
+local item_width_real = frame_inner_width / self.n_pos - 2*item_border_size
+local item_width = math.ceil(item_width_real)
+local item_width_adjust = item_width - item_width_real
```
and name your `diff` to something like `item_width_to_add`, for easier future reading.

Reply:
I like it. It's a lot more clear :)